### PR TITLE
V3.0.6rc

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Release candidate for v3.0.6, a version release primarily intended to support operational systems using pysat on an older version of Python.

Addresses # (issue)

## Type of change

- Release Candidate

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide
instructions so we can reproduce. Please also list any relevant details for
your test configuration

- Online unit test suite

**Test Configuration**:
* Operating system: GitHub
* Version number: Python 3.6, 3.9, 3.10
* Any details about your local setup that are relevant

# Checklist:

- [x] Update the version number
- [x] Verify and update Zenodo information in .zenodo.json
- [x] doc tests pass locally
- [x] activate this branch on readthedocs (https://pysat.readthedocs.io/en/v3-0-6rc/)
- [x] Upload the test distribution to the testPyPi server
- [x] Contacted developers and users about the future release
- [x] Performed integration tests with downstream packages (include list of these packages)
- [x] Create a wiki page with the release text (https://github.com/pysat/pysat/wiki/v3.0.6-Release-Text)
- [x] After approval, confirm all changes from the RC pull are listed in the wiki release text.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -26,16 +26,18 @@ classifiers =
...
   Operating System :: MacOS :: MacOS X
   Operating System :: POSIX :: Linux
+  Operating System :: Microsoft :: Windows
 license_file = LICENSE
 long_description = file: README.md
 long_description_content_type = text/markdown
 
 [options]
-python_requires = >= 3.5
+python_requires = >= 3.6.8
```
Comment (aburrell) [inline setup.cfg:40]:
```suggestion
python_requires = >= 3.6
```

I recommend making this simple, even though we test the particular version I needed to work.

**Comment by rstoneback** [comment]
I updated the release text with the changelog from the CI pull.

**Comment by rstoneback** [comment]
Locally ran test against:
 - [x] pysatSpaceWeather RC
 - [ ] pysatModels (in progress against develop)
 - [ ] pysatNASA
 - [ ] pysatCDAAC
 - [ ] pysatMissions
 - [ ] pysatSeasons
 - [ ] pysatMadrigal

**Comment by jklenzing** [comment]
Note that numpy dropped a new version a few days ago that is causing issues in the Github Actions elsewhere (pysatNASA, maybe more)

**Comment by rstoneback** [comment]
> Note that numpy dropped a new version a few days ago that is causing issues in the Github Actions elsewhere (pysatNASA, maybe more)

Bummer, but thanks for the update. pysatMadrigal tests are still running but after that I think this RC is fully ready for review and whatnot.

**Comment by rstoneback** [comment]
Ran into a small download issue from pysatMadrigal and dmsp ivm f11 with the unit tests.
```
FAILED pysatMadrigal/tests/test_instruments.py::TestInstruments::test_remote_file_list[inst_dict1] - ValueError: unable to open url http://cedar.openmadrigal.org/getExperimentsService.py?code=8100&startyear=1998&startmonth=...
```
Ran the download myself and things worked fine, including loading data. Think it was just a typical connection misfire.

**Review by aburrell — APPROVED** [review]

**Pull request merged**